doc/string.rdoc

```diff
@@ -48,6 +48,42 @@ class String
     Primitive.rb_str_init(str, encoding, capacity)
   end
 
+  #  call-seq:
+  #    bytes -> array_of_bytes
+  #
+  #  Returns an array of the 8-bit bytes in +self+:
+  #
+  #    'hello'.bytes # => [104, 101, 108, 108, 111]
+  #    'тест'.bytes  # => [209, 130, 208, 181, 209, 129, 209, 130]
+  #    'こんにちは'.bytes
+  #    # => [227, 129, 147, 227, 130, 147, 227, 129, 171, 227, 129, 161, 227, 129, 175]
+  #
+  def bytes; end
```

Comment:
I should probably have brought this up earlier, but moving the documentation out of the source (`string.c`) makes the `Toggle source` feature of RDoc useless. I think we should find a way to keep that feature functional. I see @duerst brought this up many years ago in ruby/rdoc#409. Has there been any progress on that issue?

Reply:
> I should probably have brought this up earlier, but moving the documentation out of the source (`string.c`) makes the `Toggle source` feature of RDoc useless. I think we should find a way to keep that feature functional. I see @duerst brought this up many years ago in [ruby/rdoc#409](https://github.com/ruby/rdoc/issues/409). Has there been any progress on that issue?

I think it's important to be able to show non-US-ASCII in the examples, especially for classes such as String and Encoding.  I'm thinking that *many* methods's documentation, in String, Encoding, and perhaps others, would profit from examples outside of US-ASCII.

So the issue, as I see it, is whether such examples should wait for an RDoc resolution, or we should go ahead and put them into the C code as we go.  Does this need to go to the developers' meeting?

Reply:
I absolutely agree that we should have non-ASCII examples. We should think of a solution that doesn't break any of the existing features of RDoc. I won't let this issue block you, just something for us to think about in parallel.

Reply:
@peterzhu2118, this is ready, then?


After this merge, I'll make include files for a couple of String methods; then we can get some comments from other committers?

Reply:
I just put in a few more comments